### ISSUE TITLE
cannot build i18n-embed-fl due to dashmap version

### ISSUE THREAD

**Issue by Roguelazer** [kellpossible/cargo-i18n#55]
A routine `cargo update` broke i18n-embed-fl today; it looks like `src/lib.rs` calls the `insert_and_get` method on a DashMap instance, but that method was removed in dashmap 4.0.0.  For some reason, `cargo update` believes that `4.0.0-rc6` and `4.0.0` are compatible versions, so it freely upgrades between them.

I think the easiest fix would be to just move to the `dashmap` 4.0.0 final API and bump the dependency...

**Comment by str4d** [comment]
If anyone else is blocking on this, you can temporarily pin the old version of `dashmap` by adding an explicit dependency to your `Cargo.toml`:

```
[dependencies]
i18n-embed-fl = "0.2"
# Temporarily pin until https://github.com/kellpossible/cargo-i18n/pull/56 is released.
dashmap = "=4.0.0-rc6"
```

**Comment by kellpossible** [comment]
Hi guys, is this problem now fixed for you in `i18n-embed-fl` version `0.3.1`?

**Comment by Roguelazer** [comment]
Looks good to me!
